SpatialGDK/Source/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -333,26 +332,36 @@ int64 USpatialActorChannel::ReplicateActor()
 	{
 		FOutBunch DummyOutBunch;
 
-		FClassInfo* ClassInfo = NetDriver->TypebindingManager->FindClassInfoByClass(Actor->GetClass());
-
 		for (UActorComponent* ActorComponent : Actor->GetReplicatedComponents())
 		{
-			if (ClassInfo->SubobjectClasses.Contains(ActorComponent->GetClass()))
+			FUnrealObjectRef ObjectRef = NetDriver->PackageMap->GetUnrealObjectRefFromObject(ActorComponent);
+
+			if (ObjectRef != SpatialConstants::NULL_OBJECT_REF)
 			{
-				bWroteSomethingImportant |= ReplicateSubobject(ActorComponent, RepFlags);
+				FClassInfo& SubobjectInfo = *Info->SubobjectInfo[ObjectRef.Offset];
```

Comment:
Nit: we're passing a pointer into `ReplicateSubobject` anyway, so why dereference it here?